src/presentation/Answer.ts

```diff
@@ -100,21 +104,30 @@ export class Answer {
             this.markTheAnswers(quizMaster, collectedAnswers);
             this.colourScoreBar(quizMaster, collectedAnswers);
 
-            /* User get a note if some answers are missing */
-            if (questions.type === QuestionType.MULTIPLE_CHOICE) {
-                if (!quizMaster.evaluateAnswers(questions, collectedAnswers)) {
-                    let numberOfMissingAnswers = quizMaster.numberOfMissingAnswers(questions, collectedAnswers);
-                    let givenAnswersLength = questions.answers.length;
-                    if (1 < numberOfMissingAnswers && 1 < givenAnswersLength) {
-                        let hintForMissingAnswers = document.getElementById("missingAnswers") as HTMLParagraphElement;
-                        hintForMissingAnswers.textContent = numberOfMissingAnswers.toString() + ' answers missing';
-
-                        addButton.showNote();
-                        addButton.closeNote();
-                    }
+            this.noteForMultipleChoice(quizMaster, collectedAnswers);
+
+            quizMaster.handleQuizScore(collectedAnswers);
+        }
+    };
+
+    /** User get a note if some answers are missing */
+    noteForMultipleChoice(quizMaster: QuizMaster, collectedAnswers: string[]) {
```

Comment:
Schön, dass du es rausgenommen hast :-)

Reply:
ja wurde jetzt nötig. hatte einen Logikfehler in einer Methode gefunden und konnte den Fehler schwer finden